<launch>
  	<node name="train_data_node" pkg="automatic_painting" type="train_data_node" 
       output="screen" >
		<param name="data_file" value="output-features.csv" />
		<param name="save_file" value="data.xml" />
		<param name="num_samples" value="431" />
	</node>
</launch>
